pkgs/development/libraries/science/math/cutensor/default.nix

```diff
@@ -14,8 +14,12 @@ rec {
   };
 
   cutensor_cudatoolkit_10_2 = cutensor_cudatoolkit_10_1.override {
+    version = "1.3.1.3";
     libPath = "lib/10.2";
     cudatoolkit = cudatoolkit_10_2;
+    # 1.3.1 is compatible with CUDA 11.0, 11.1, and 11.2:
+    # ephemeral doc at https://developer.nvidia.com/cutensor/downloads
+    sha256 = "sha256-mNlVnabB2IC3HnYY0mb06RLqQzDxN9ePGVeBy3hkBC8=";
```

Comment:
Why is this necessary? It doesn't seem to be affected by changing `cudnn_cudatoolkit_11` or `cutensor_cudatoolkit_11`

Reply:
I don't understand, can you explain how you change cuddn or cutensor?
On top of this branch building cutensor with cudatoolkit 11 depends on this version of cutensor.
```
$ nix-build -A cutensor_cudatoolkit_11
these 2 derivations will be built:
  /nix/store/b9wiirqsjqzcfjps14iyvlpivrd15mlk-cudatoolkit-11.4.1.drv
  /nix/store/5lhpy515j0cqklpma7kjphl9hilksniw-cudatoolkit-11.4-cutensor-1.3.1.3.drv
[...]
```

Reply:
But the edit here is for `cutensor_cudatoolkit_10_2` not `cutensor_cudatoolkit_11`, right?

Reply:
We would typically bump cutensor globally, but cutensor 1.3 drops support for 10.1 so we had to keep 10.1 as-is and bump only 10.2 onward: https://github.com/NixOS/nixpkgs/pull/137553#discussion_r716916285

Reply:
Ok, I'm a little confused, but I also haven't poked around with this code much so I trust your judgement.